KIPs/kip-82.md

```diff
@@ -0,0 +1,334 @@
+---
+kip: 82
+title: A new GC reward structure due to abolition of the Gini coefficient
+author: Yeri<yeriel.lee@krustuniverse.com>, Daniel<daniel.cc@krustuniverse.com>, Aidan<aidan.kwon@krustuniverse.com>, Ollie<ollie.j@krustuniverse.com>, Sam<sam.seo@krustuniverse.com>, Uno<uno.lee@krustuniverse.com>, Eddie<eddie.kim0@krustuniverse.com>
+discussion-to: https://govforum.klaytn.foundation/t/proposal-a-new-gc-reward-structure-to-abolition-of-the-gini-coefficient/21
+status: Draft
+type: Standards Track
+category: Core
+created: 2022-09-21
+---
+
+
+## Simple Summary
+A renewal of the Governance Council(GC) reward structure due to abolition of the Gini coefficient.
+
+## Abstract
+In addition to traditional enterprises, Klaytn is expanding the Governance Council (GC) by bringing DAOs in response to the growth of nontraditional entities. Such a change in the GC ecosystem will restructure the entire Klaytn governance structure. Abolishing the current block proposal selection method based on the Gini coefficient and staking quantity, the new reward program alleviates the problem of providing insufficient compensation for the network contribution and block verification due to lack of staking. The instability of certain nodes with high staking amounts can result in poor network stability as a whole. The renewed structure enables to reward the staking amount regardless of block miner by providing 20% total rewards for block generation and 80% based on staking amount. This form is to reward the participants for providing network stability with node operation and financial contribution.
```

Comment:
Since the reward ratios, such as 20% for the block proposer reward and 80% for the staking reward, might need to be changed in the future depending on the node operation cost, it would be good to have the reward ratios as governance parameters so GC can change them when necessary. @yeri-lee @aidan-kwon @blukat29 What do you think?

Reply:
I'd like to suggest to have a separate paragraph to describe these numbers, 20% and 80%, and to mention that they could be changed in the future by GC voting and they are used in this KIP as an initial proposal.

Reply:
The 20/80 ratio is indeed a governance parameter. It just wasn't highlighted in the document.

```
reward.usekip82: bool = false
reward.kip82ratio: string = "20/80"
```

I agree it's worth mentioning that the ratio is tentative and can be changed by GC governance.

Reply:
Added 6cfa539.